assets/install-rhcos.sh

```
@@ -0,0 +1,52 @@
+
+dd if=/dev/zero of=$device bs=1M count=1 status=progress
+
+coreos-installer install $device \
+  --append-karg "console=hvc0 console=ttyAMA0 earlycon=pl011,0x13010000 ignore_loglevel" \
+  --ignition-file /tmp/bf.ign \
+  --offline
+
+sync
+
+if efibootmgr -v | grep -q "Red-Hat CoreOS GRUB"; then
+  efibootmgr -b $(efibootmgr -v | grep "Red-Hat CoreOS GRUB" | awk '{print $1}' | cut -d' ' -f1) -B
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)